### PR TITLE
Fixed dtype error in `se3.py` that came up in unit tests

### PR DESCRIPTION
**Pull request by joeaortiz**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by luisenp — APPROVED** [review]
Thanks for the fix! LGTM, just curious if `where` wouldn't accept a scalar tensor rather than having to call `full`.

**Review by mhmukadam — APPROVED** [review]

**Comment by fantaosha** [comment]
> Thanks for the fix! LGTM, just curious if `where` wouldn't accept a scalar tensor rather than having to call `full`.

That's also what I thought. How about 
`torch.tensor(-1/12.0, dtype=sine_by_theta.dtype, device=sine_by_theta.device)`.

File: theseus/geometry/se3.py [hunk 1]
```diff
@@ -285,14 +285,26 @@ def exp_map(
 
             jac[:, 3:, 3:] = jac[:, :3, :3]
 
+            minus_one_by_twelve = torch.full(
+                near_zero.shape,
+                -1 / 12.0,
+                dtype=sine_by_theta.dtype,
+                device=sine_by_theta.device,
+            )
```
Comment (fantaosha) [inline theseus/geometry/se3.py:293]: I just tested.
<pre><code>minus_one_by_twelve = torch.tensor(-1 / 12.0, 
                                   dtype=sine_by_theta.dtype, 
                                   device=since_by_theta.device)</code></pre>
should work for `torch.where()`.

File: theseus/geometry/se3.py [hunk 2]
```diff
@@ -285,14 +285,26 @@ def exp_map(
...
-                -1 / 12.0,
+                minus_one_by_twelve,
                 (sine_by_theta - 2 * one_minus_cosine_by_theta2) / theta2_nz,
             )
+            minus_one_by_sixty = torch.full(
+                near_zero.shape,
+                -1 / 60.0,
+                dtype=one_minus_cosine_by_theta2.dtype,
+                device=one_minus_cosine_by_theta2.device,
+            )
```
Comment (fantaosha) [inline theseus/geometry/se3.py:304]: Same here.

**Review by fantaosha — CHANGES_REQUESTED** [review]

**Review by fantaosha — APPROVED** [review]

**Pull request merged**